dll/win32/browseui/lang/es-ES.rc

```diff
@@ -99,13 +105,13 @@ IDM_POPUPMENU MENUEX DISCARDABLE
 BEGIN
     POPUP ""
     BEGIN
-        POPUP "&View", IDM_VIEW_MENU
+        POPUP "&Ver", IDM_VIEW_MENU
         BEGIN
-            MENUITEM "&Large Icons", IDM_LARGE_ICONS
-            MENUITEM "&Small Icons", IDM_SMALL_ICONS
+            MENUITEM "Íconos &grandes", IDM_LARGE_ICONS
```

Comment:
In Spain Spanish we generally use "Icono" rather than "Ícono", at least it sounds weird to me with the strong syllable being the first one.

Reply:
The RAE guys says that this word is an "esdrújula", so the correct pronunciation is the I with accent.

Reply:
Then I have to disagree with the RAE, I guess. Keep it, it just sounds horrible to me. :P

Reply:
According with http://lema.rae.es/dpd/srv/search?key=icono both you and me were right 😃 "ícono" as esdrújula is used in latam, and icono is used in Spain, is the same case as "computadora" and "ordenador"